### ISSUE TITLE
Question about The training parameter For SimplerEnv

### ISSUE THREAD

**Issue by LukeLIN-web** [Robot-VLAs/RoboVLMs#29]
APPENDIX C
IMPLEMENTATION DETAILS
a) Hyper-parameters and Training Details
We set the global batch size as 128 and the warmup ratio is 0.25 epoch (5K steps for
OpenX Embodiment pre-train). All models included in this paper are trained on a cluster of 4 x 8 A100 GPUs.

But 
b) Checkpoint selection.: said  
 For SimplerEnv, we train the model for 100K iterations with a batch size of 512
truncated trajectories and report the best-performing model with a 10K-iteration training interval. 

Why they are different?

**Comment by XXXUAN0904** [comment]
> APPENDIX C IMPLEMENTATION DETAILS a) Hyper-parameters and Training Details We set the global batch size as 128 and the warmup ratio is 0.25 epoch (5K steps for OpenX Embodiment pre-train). All models included in this paper are trained on a cluster of 4 x 8 A100 GPUs.
> 
> But b) Checkpoint selection.: said For SimplerEnv, we train the model for 100K iterations with a batch size of 512 truncated trajectories and report the best-performing model with a 10K-iteration training interval.
> 
> Why they are different?

Hi！Could you show me your tensorboard train_loss image as well as the val_loss image and the config file, I'm having some problems with the finetune bridge, thanks a lot!

**Comment by lixinghang12** [comment]
Thanks for your correction, we utilize 512 as batch size for SimplerEnv training, we would fix the error in the table
